src/rulesets/oas/index.json

```diff
@@ -515,7 +515,15 @@
       "recommended": true,
       "formats": ["oas3"],
       "type": "style",
-      "given": ["$.paths..examples[*]", "$.components..examples[*]"],
+      "given": [
+        "$.components.examples[*]",
+        "$.paths[*][*]..content[*].examples[*]",
+        "$.components.content[*].examples[*]",
```

Comment:
I'm not sure what `$.components.content[*]` is, these are the only valid properties in components.

<img width="865" alt="image" src="https://user-images.githubusercontent.com/67381/103307541-9bdfce00-4a10-11eb-82db-4706e0fb0f5d.png">

Reply:
FWIW, I simply reverted the code to the previous version of it and applied a few additional changes. I simplified the paths for this rule some time ago and I made them tad too general.
I actually saw that table a few mins ago, since I wasn't sure whether `examples` is a valid property, yet didn't verify `content`. The old code used to have it, but not sure why.
I'll get rid of it then.